Resources/Prototypes/GameRules/events.yml

```
@@ -241,17 +241,17 @@
     maxDuration: 120
   - type: PowerGridCheckRule
 
-# - type: entity # DeltaV - Removes the random sentience.
-#   id: RandomSentience
-#   parent: BaseGameRule
-#   noSpawn: true
-#   components:
-#   - type: StationEvent
-#     weight: 10
-#     duration: 1
-#     startAudio:
-#       path: /Audio/Announcements/attention.ogg
-#   - type: RandomSentienceRule
+- type: entity
+  id: RandomSentience
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    weight: 10
```

Just going to say this again; this is an insanely 0RP event, vending machines talking isn't as fun as one imagined an having this take up and event slot is BAD.

This needs to have lower weight.
```suggestion
    weight: 1
```